apps/dashboard/app/views/dashboard/_motd_osc.html.erb

```diff
@@ -1,18 +1,15 @@
-<h3><%= @motd.title %></h3>
-
-<% if @motd.messages.each do |m| %>
-
-  <hr>
-  <div class="motd">
-    <h4 class="motd_title"><%= m.date %> - <%= m.title %></h4>
-    <div class="motd_body"><p><%=raw markdown( m.body ) %></p></div>
-  </div>
-
-<% end.empty? %>
-
-  <hr>
-  <div class="motd">
-    <h4 class="motd_title">There are no new messages.</h4>
-  </div>
-
+<%= content_tag(:div, id: 'motd-container') do %>
+  <%= tag.h3 @motd.title %>
```

Comment:
Instead of using content_tag and tag.h3 in other places we have typically used literal html instead:

```
<div id="motd-container">
  <h3><%= @motd.title %></h3>
  
  ...
  ```
  
It is more text, we have to deal with end tags, but it is faster to read since we are all so conditioned to read HTML.